speed-dreams/source-2.2.3/src/modules/racing/standardgame/racesituation.cpp

```diff
@@ -501,8 +500,8 @@ int ReSituationUpdater::threadLoop()
             GfProfStopProfile("reOneStep*");
 
             // Send car physics to network if needed
-            if (NetGetNetwork())
-                NetGetNetwork()->SendCarControlsPacket(pCurrReInfo->s);
+            /*if (NetGetNetwork())
+                NetGetNetwork()->SendCarControlsPacket(pCurrReInfo->s);*/
```

Comment:
Why is this commented out, it doesn't seem relevant to the PR? It works fine on my system with just leaving these 2 lines of code in. If we really want to remove all network stuff, we should do that in a specific story.

Reply:
This was for the internal server of speed dreams, it was crashing because the module was removed. I will put it back